data/typelibrary/utils-1.0.0/typelib/splitting/WORD_TO_BOOLS.fct

```diff
@@ -0,0 +1,104 @@
+	</Identification>
+	<VersionInfo Version="1.0" Author="franz" Date="2024-02-22">
+	</VersionInfo>
+	<CompilerInfo>
+	</CompilerInfo>
+	<InterfaceList>
+		<EventInputs>
+			<Event Name="REQ" Type="Event" Comment="Normal Execution Request">
+				<With Var="IN"/>
+			</Event>
+		</EventInputs>
+		<EventOutputs>
+			<Event Name="CNF" Type="Event" Comment="Execution Confirmation">
+				<With Var="OUT_00"/>
+				<With Var="OUT_01"/>
+				<With Var="OUT_02"/>
+				<With Var="OUT_03"/>
+				<With Var="OUT_04"/>
+				<With Var="OUT_05"/>
+				<With Var="OUT_06"/>
+				<With Var="OUT_07"/>
+				<With Var="OUT_08"/>
+				<With Var="OUT_09"/>
+				<With Var="OUT_10"/>
+				<With Var="OUT_11"/>
+				<With Var="OUT_12"/>
+				<With Var="OUT_13"/>
+				<With Var="OUT_14"/>
+				<With Var="OUT_15"/>
+			</Event>
+		</EventOutputs>
+		<InputVars>
+			<VarDeclaration Name="IN" Type="WORD" Comment="WORD Input"/>
+		</InputVars>
+		<OutputVars>
+			<VarDeclaration Name="OUT_00" Type="BOOL" Comment="BIT 00"/>
+			<VarDeclaration Name="OUT_01" Type="BOOL" Comment="BIT 01"/>
+			<VarDeclaration Name="OUT_02" Type="BOOL" Comment="BIT 02"/>
+			<VarDeclaration Name="OUT_03" Type="BOOL" Comment="BIT 03"/>
+			<VarDeclaration Name="OUT_04" Type="BOOL" Comment="BIT 04"/>
+			<VarDeclaration Name="OUT_05" Type="BOOL" Comment="BIT 05"/>
+			<VarDeclaration Name="OUT_06" Type="BOOL" Comment="BIT 06"/>
+			<VarDeclaration Name="OUT_07" Type="BOOL" Comment="BIT 07"/>
+			<VarDeclaration Name="OUT_08" Type="BOOL" Comment="BIT 08"/>
+			<VarDeclaration Name="OUT_09" Type="BOOL" Comment="BIT 09"/>
+			<VarDeclaration Name="OUT_10" Type="BOOL" Comment="BIT 10"/>
+			<VarDeclaration Name="OUT_11" Type="BOOL" Comment="BIT 11"/>
+			<VarDeclaration Name="OUT_12" Type="BOOL" Comment="BIT 12"/>
+			<VarDeclaration Name="OUT_13" Type="BOOL" Comment="BIT 13"/>
+			<VarDeclaration Name="OUT_14" Type="BOOL" Comment="BIT 14"/>
+			<VarDeclaration Name="OUT_15" Type="BOOL" Comment="BIT 15"/>
+		</OutputVars>
+	</InterfaceList>
+	<FunctionBody>
+		<ST><![CDATA[FUNCTION WORD_TO_BOOLS
+
+VAR_INPUT
+	IN : WORD;    // WORD Input
+END_VAR
+
+VAR_OUTPUT
+	OUT_00 : BOOL;    // BIT 00
+	OUT_01 : BOOL;    // BIT 01
+	OUT_02 : BOOL;    // BIT 02
+	OUT_03 : BOOL;    // BIT 03
+	OUT_04 : BOOL;    // BIT 04
+	OUT_05 : BOOL;    // BIT 05
+	OUT_06 : BOOL;    // BIT 06
+	OUT_07 : BOOL;    // BIT 07
+	OUT_08 : BOOL;    // BIT 08
+	OUT_09 : BOOL;    // BIT 09
+	OUT_10 : BOOL;    // BIT 10
+	OUT_11 : BOOL;    // BIT 11
+	OUT_12 : BOOL;    // BIT 12
+	OUT_13 : BOOL;    // BIT 13
+	OUT_14 : BOOL;    // BIT 14
+	OUT_15 : BOOL;    // BIT 15
+END_VAR
+
+(* this Function extracts the 16 BOOLS from a word *)
```

Comment:
this should go at the top of the function

Reply:
you mean top of the Function, also Top of the Variable Definition, right ?

Reply:
this is done @MoritzO02. good.